pkgs/applications/science/electronics/csxcad/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, fparser
+, tinyxml
+, hdf5
+, cgal_5
+, vtk
+, boost
+, gmp
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "csxcad";
+  version = "0.6.2";
```

```suggestion
  version = "unstable-2020-02-08";
```

since the revision points to a commit of that date

---

Yes, you are correct. Thanks for catching that. It is now fixed.